user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -278,6 +291,13 @@ def define_invalid_arg_cases(self):
                 [ArgValueCase.VALUE_A, ArgValueCase.VALUE_A, ArgValueCase.IGNORE]
             ]
         }
+        self.rejected['Cluster By Properties Cannot go with OnPrem'] = {
+            'valid': False,
+            'callable': partial(self.validate_onprem_with_cluster_props),
+            'cases': [
+                [ArgValueCase.VALUE_A, ArgValueCase.VALUE_C, ArgValueCase.UNDEFINED]
```

Comment:
Does `VALUE_C` corresponds to auto-tuner file, but the description says "By Properties", which seems to refer to `VALUE_B`?